lint/file.go

```diff
@@ -126,8 +128,13 @@ type enableDisableConfig struct {
 	position int
 }
 
-func (f *File) disabledIntervals(rules []Rule) disabledIntervalsMap {
-	re := regexp.MustCompile(`^//[\s]*revive:(enable|disable)(?:-(line|next-line))?(?::([^\s]+))?[\s]*$`)
+const directiveRE = `^//[\s]*revive:(enable|disable)(?:-(line|next-line))?(?::([^\s]+))?[\s]*(?: (.+))?$`
+const directivePos = 1
+const modifierPos = 2
+const rulesPos = 3
+const reasonPos = 4
+func (f *File) disabledIntervals(rules []Rule, specifyDisableReason bool) disabledIntervalsMap {
+	re := regexp.MustCompile(directiveRE)
```

Comment:
Maybe we can also compile outside of the function since we're not changing the regex?

Reply:
Done